rts/lra/lra-test/lra-test-arquillian-extension/src/main/java/io/narayana/lra/arquillian/spi/NarayanaLRARecovery.java

```diff
@@ -0,0 +1,73 @@
+package io.narayana.lra.arquillian.spi;
+
+import io.narayana.lra.LRAConstants;
+import io.narayana.lra.logging.LRALogger;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+import java.net.URI;
+
+public class NarayanaLRARecovery {
+
+    public void waitForCallbacks(URI lraId) {
+        // no action is needed, tck callbacks calls are sufficiently fast for TCK
+    }
+
+    // TODO remove once this class implements LRARecoveryService
```

Comment:
Can a JBTM be created for the TODO?

Reply:
I see, I think something is necessary such that this comment will get removed when LRA branch was merged into master. Maybe we need to rebase LRA branch on master and add a commit removing this line (associating the commit with JBTM-3204/3221) after this PR could be merged?

Reply:
@tomjenkinson this is exactly my plan. After this PR is merged, I will have to merge master into LRA so I can build forget calls for nested participants ([JBTM-3228](https://issues.redhat.com/browse/JBTM-3228)) on top of this solution.